WcaOnRails/public/404.html

```diff
@@ -1,67 +1,119 @@
 <!DOCTYPE html>
-<html>
-<head>
-  <title>The page you were looking for doesn't exist (404)</title>
-  <meta name="viewport" content="width=device-width,initial-scale=1">
-  <style>
-  body {
-    background-color: #EFEFEF;
-    color: #2E2F30;
-    text-align: center;
-    font-family: arial, sans-serif;
-    margin: 0;
-  }
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<link href="http://fonts.google.apis.com/css?family=" rel="stylesheet" type="text/css" />
+		<link href="/assets/wca-0e38e06e4765b5d1c9c349f9660ae864f509c331e281c1e617e8542cf32a48b3.ico" rel="icon" type="image/x-icon" />
```

Comment:
I don't think this is a great idea. Is this needed? Can you just link directly to https://github.com/cubing/worldcubeassociation.org/blob/master/WcaOnRails/public/favicon.ico?